benchmark/benchmarks.jl

```diff
@@ -34,57 +42,64 @@ for subdir ∈ subdirs
 
     A = MatrixMarket.mmread(joinpath(iterpath, "K_0.mtx"))
     b = readdlm(joinpath(iterpath, "rhs_0.rhs"), Float64)[:]
-    nnz_A = nnz(tril(A))
-    if size(A, 1) ≤ 25000
+    nnz_A = nnz(tril(A, -1))
+    n = size(A,1)
+    if n ≤ 25000
       name = "$(subdir)_$(formulation)"
       push!(names_sqd, name)
-      println(name)
 
       p_amd      = amd(A)
       p_symamd   = symamd(A)
       p_metis, _ = Metis.permutation(A)
       p_symrcm   = symrcm(A)
-      p_classic  = collect(1:size(A,1))
+      p_classic  = collect(1:n)
+      println(name)
+      
+      # ldlt_amd     = ldl(A, p_amd)
+      # ldlt_symamd  = ldl(A, p_symamd)
+      # ldlt_metis   = ldl(A, p_metis)
+      # ldlt_symrcm  = ldl(A, p_symrcm)
+      # ldlt_classic = ldl(A, p_classic)
 
-      ldlt_amd     = ldl(A, p_amd)
-      ldlt_symamd  = ldl(A, p_symamd)
-      ldlt_metis   = ldl(A, p_metis)
-      ldlt_symrcm  = ldl(A, p_symrcm)
-      ldlt_classic = ldl(A, p_classic)
+      # push!(nnz_sqd, nnz_A)
+      # push!(ratio_amd, nnz(ldlt_amd) / nnz_A)
+      # push!(ratio_symamd, nnz(ldlt_symamd) / nnz_A)
+      # push!(ratio_metis, nnz(ldlt_metis) / nnz_A)
+      # push!(ratio_symrcm, nnz(ldlt_symrcm) / nnz_A)
+      # push!(ratio_classic, nnz(ldlt_classic) / nnz_A)
```

Comment:
I think all this commented out stuff can be removed now actually. Your suite is much better.